Src/functions.c

```diff
@@ -141,3 +141,31 @@ void gpio_mode_QUICK(gpio_type* gpio_periph, uint32_t mode,
 }
 #endif
 #endif
+
+
+#ifdef WCH
+
+void delayMicros(uint32_t micros)
+{
```

Comment:
i'm not a fan of the busy loop method, depends too much on the clock selected. Works for current v203 but will fail on other variants. Don't we have a timer we can use?

Reply:
All timers have been used

Reply:
Originally, this delay should use the core timer, so that it can be used as long as it is a chip with the same core. However, the V203 has a total of 4 peripheral timers.   

1 for scheduled task, 2 for commutation tasks, 1 for dshot signal detection task, and 1 for PWM output task，so.....